lib/capybara/spec/session/visit_spec.rb

```diff
@@ -17,6 +17,17 @@
     expect(@session).to have_content('Another World')
   end
 
+  # TODO: This test passes on racktest and selenium - although I think all it's really testing
+  # is that scheme defaults to http if not specified as anything else
+  # it "should fetch a response when absolute URI doesn't have a scheme", focus: true do
+  #   # Preparation
+  #   @session.visit('/')
+  #   root_uri = URI.parse(@session.current_url)
+  #
+  #   @session.visit("//#{root_uri.host}:#{root_uri.port}/foo")
```

Comment:
Exactly -- thats not what visit does, but really all that test is checking is that scheme defaults to 'http' -- which it does because Capybara.app_host or the server uri in visit is set to a scheme of http.  If app_host was not set and no server was started then it would fail if a scheme wasn't specified - or potentially default to a scheme of :file  (as per Jonas' comment on @dstosik issue)